common/components/ui/Modal.jsx

```diff
@@ -0,0 +1,107 @@
+// @flow
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import closeIcon from 'assets/images/icon-x.svg';
+
+import './Modal.scss';
+
+export default class Modal extends Component {
+  static propTypes = {
+    isOpen: PropTypes.bool,
+    size: PropTypes.oneOf(['small', 'medium', 'large', 'xlarge']),
```

Comment:
I am not even sure what `max-width: 600px; width: 100%;` does. 

But if you have chidren wrapped in container with maxwidth it is exact same behaviour as seen in modal right now or not?